upcloud_api/constants.py

```diff
@@ -31,11 +31,10 @@ class OperatingSystems(object):
         'Debian 8.11': '01000000-0000-4000-8000-000020030100',
         'Debian 9.9': '01000000-0000-4000-8000-000020040100',
         'Debian 10.0': '01000000-0000-4000-8000-000020050100',
-        'Ubuntu 14.04': '01000000-0000-4000-8000-000030040200',
```

Comment:
Does the UpCloud API give an error if you try/tried to get this? Just asking in case someone is still running this script with Ubuntu 14.04 (or Windows 2012) and if it's actually working 🤔

Reply:
Ubuntu 14.04 is no longer available. API gives 404 for this storage, so it can't be used anymore.